android-sdk/src/main/java/org/sagebionetworks/bridge/android/util/retrofit/BodyOnSubscribe.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 Jake Wharton
```

Comment:
didn't he write retrofit?  seems odd code like this wouldn't just be in the library